[2024-08-26T23:21:20.922+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-26T23:21:20.962+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nyc_dag.get_timestamp manual__2024-08-26T23:21:16.994161+00:00 [queued]>
[2024-08-26T23:21:20.980+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nyc_dag.get_timestamp manual__2024-08-26T23:21:16.994161+00:00 [queued]>
[2024-08-26T23:21:20.981+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 0
[2024-08-26T23:21:21.013+0000] {taskinstance.py:2879} INFO - Executing <Task(BashOperator): get_timestamp> on 2024-08-26 23:21:16.994161+00:00
[2024-08-26T23:21:21.022+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=218) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-26T23:21:21.024+0000] {standard_task_runner.py:72} INFO - Started process 219 to run task
[2024-08-26T23:21:21.025+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'nyc_dag', 'get_timestamp', 'manual__2024-08-26T23:21:16.994161+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/nyc_dag_script.py', '--cfg-path', '/tmp/tmpwczfnxmm']
[2024-08-26T23:21:21.026+0000] {standard_task_runner.py:105} INFO - Job 3: Subtask get_timestamp
[2024-08-26T23:21:21.121+0000] {task_command.py:467} INFO - Running <TaskInstance: nyc_dag.get_timestamp manual__2024-08-26T23:21:16.994161+00:00 [running]> on host 1371581a889e
[2024-08-26T23:21:21.268+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='tolutosin16@gmail.org' AIRFLOW_CTX_DAG_OWNER='tolu_samuel' AIRFLOW_CTX_DAG_ID='nyc_dag' AIRFLOW_CTX_TASK_ID='get_timestamp' AIRFLOW_CTX_EXECUTION_DATE='2024-08-26T23:21:16.994161+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-26T23:21:16.994161+00:00'
[2024-08-26T23:21:21.270+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-26T23:21:21.341+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-26T23:21:21.342+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo current date adn time is $(date)']
[2024-08-26T23:21:21.365+0000] {subprocess.py:86} INFO - Output:
[2024-08-26T23:21:21.376+0000] {subprocess.py:93} INFO - current date adn time is Mon Aug 26 23:21:21 UTC 2024
[2024-08-26T23:21:21.378+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-26T23:21:21.430+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-26T23:21:21.431+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=nyc_dag, task_id=get_timestamp, run_id=manual__2024-08-26T23:21:16.994161+00:00, execution_date=20240826T232116, start_date=20240826T232120, end_date=20240826T232121
[2024-08-26T23:21:21.483+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-26T23:21:21.515+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-26T23:21:21.521+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
